.github/workflows/pipeline-k8s-check.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow performs static code analysis of your Kubernetes object definitions
+# using kube-score and kube-linter tools.
+name: Kube Checks
+
+on:
+  workflow_call:
+    inputs:
+      kube_lint_target:
+        description: 'The directory to run kube-linter'
+        required: false
+        default: "./"
+        type: string
+      kube_score_target:
+        description: 'The yaml or yaml files to run kube-score on'
+        required: false
+        default: "./**/*.y*ml"
+        type: string
+
+jobs:
+  kube-checks:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Kube-Score
+        run: |
+          curl -LO https://github.com/zegl/kube-score/releases/download/v1.18.0/kube-score_1.18.0_linux_amd64.tar.gz
+          tar -xzf kube-score_1.18.0_linux_amd64.tar.gz
+          mv kube-score /usr/local/bin/

```

Comment:
I would add a sanity check here `kube-score version`

Reply:
since I use a plugin from marketplace, no need for that.